src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/service/SynchronisationQueueService.kt

```diff
@@ -33,6 +34,9 @@ class SynchronisationQueueService(
       SendMessageRequest.builder()
         .queueUrl(queue.queueUrl)
         .messageBody(sqsMessage.toJson())
+        .messageAttributes(
+          mapOf("eventType" to MessageAttributeValue.builder().dataType("String").stringValue("prisoner-from-nomis-synchronisation-$messageType").build()),
```

Comment:
I'm confused as to the difference between this message and the one from the migration queue service - are they both synchronisation messages?

Reply:
They will go to different queues though the scenario will be similar.
1) An error create a mapping while running a bulk migration
2) An error creating a mapping (and other retries) while processing a NOMIS xtag event

Reply:
turns out the migration one was never used, so have removed it